crates/sncast/src/helpers/braavos.rs

```diff
@@ -77,12 +80,58 @@ where
         self.block_id
     }
 
-    async fn sign_deployment(
+    async fn sign_deployment_v1(
```

Comment:
This method is exactly the same as the one below, with the deployment type only being different. Can't we merge those two?